test/call_interpreter-test.jl

```diff
@@ -0,0 +1,14 @@
+function test_call_interpreter()
+    print("call_interpreter...")
+    Singular.libSingular.call_interpreter("ring r;")
+    result = Singular.libSingular.call_interpreter("r;")
+    @test result[1] == false
+    Singular.libSingular.call_interpreter("poly f = x;")
+    result = Singular.libSingular.call_interpreter("f;")
+    @test result[1] == false
+    @test result[2] == "x\n"
+    @test result[3] == ""
+    @test result[4] == ""
+    @test length(result) == 4
```

Comment:
Is there any way right now to interact with any of the objects created with these command? E.g. a function which takes the string "f" as input and returns the value assigned to the variable "f" right now?

Or is all interaction really meant to go through string parsing?

Reply:
Exactly, absolutely only via strings, there is never a `leftv` or anything returned.

I am actually not sure if we could have something like GAP.jl's `EvalString` that returns the value a certain command is returning. We could have this by using some dummy library function, but because of Singular's current ring I would actually consider this very hard to correctly use, and dangerous.

Reply:
We have that in SingularInterface and it is perfectly safe to use. The function provided by this PR is indeed very unsafe, and hence hard to justify. But given that we proven that it can be done better, I'd argue that's a flaw in your implementation, not Mohamed's request :-)

Reply:
I am absolutely okay with not adding this function, or adding a different one instead. But this function does what was requested, as strings by @mohamed-barakat mostly have the form `varname = something`, and do not need any returning object, but the output string. So this function should do the trick for him.

Reply:
Main problem with the function you propose is that we currently do not handle `leftv`s in Singular.jl. So the result would have to be piped through the machinery I introduced in https://github.com/oscar-system/Singular.jl/pull/104 to get something Singular.jl can actually understand. So maybe we should have both functions. You can also view this function as the beginning of a Singular REPL mode, although nobody has (and probably will ever) asked for it.